bluemira/builders/cryostat.py

```diff
@@ -119,7 +119,7 @@ def build(self) -> Component:
         return self.component_tree(
             xz=[xz_cryostat],
             xy=[self.build_xy(self.x_out)],
-            xyz=self.build_xyz(xz_cross_section),
+            xyz=self.build_xyz(xz_cross_section, degree=0),
```

Comment:
Can you open an issue about your patternable, revolvable, placeable idea as an option to replace this?

Reply:
Sure :+1:

Reply:
See [#2061](https://github.com/Fusion-Power-Plant-Framework/bluemira/issues/2061)